src/domain/customscript-identity.ts

```diff
@@ -107,7 +111,7 @@ export class CustomScriptIdentityWatchOnly extends Identity implements IdentityI
   readonly masterBlindingKeyNode: Mnemonic['masterBlindingKeyNode'];
   readonly masterPubKeyNode: Mnemonic['masterPublicKeyNode'];
   readonly namespace: CustomScriptIdentityOpts['namespace'];
-  readonly covenant: CovenantDescriptors;
+  readonly covenant: ContractTemplate;
```

Comment:
It's fun, cause we can have a Script that has NO covenant insider (ie. no introspection or timelocks in the `require` field)